### DISCUSSION TITLE
Chromium not automatically updating textboxes on webpage

### DISCUSSION THREAD

**Discussion by lecaptain64** [cefsharp/CefSharp#4297] [General]
Hi everyone.  I couldnt find an answer to my issue anywhere so I thought I would ask.  I've created a webform and used the ChromiumWebBrowser control in it.  It isnt built dynamically on form load, I simply dragged the control in a panel on the form.

The issue I'm having is when a webpage automatically is updating a textbox, Chromium is not updating it.  It stays on the same number unless you refresh the page.  Using refresh does not work well though.  This would be used in a stock market type of webpage or crypto buying page.  I'm not sure what options I may have missed or not, but when accessing the webpage from just my browser, it works as it should.  For reference, the page in question is www.kraken.com.  It is a buy and sell crypto website.  You would have to register to view the exact issue I'm having.  The original number is coming up fine, but it stays at that number.  It doesnt automatically change with a price change.  I've attached a partial screenshot of the issue I'm having. Thanks for any help that is provided.
![image](https://user-images.githubusercontent.com/117874429/200987797-d55076b8-922f-4f05-92ef-a10be282b313.png)

**Comment by amaitland** [comment]
Check the log file for errors see https://github.com/cefsharp/CefSharp/wiki/Trouble-Shooting#log-file

You can open DevTools and check the console for errors also see https://github.com/cefsharp/CefSharp/wiki/Trouble-Shooting#javascript-debugging
